src/main/scala/is/hail/stats/HWECombiner.scala

```diff
@@ -45,4 +45,19 @@ class HWECombiner extends Serializable {
   def lh = LeveneHaldane(n, nA)
 
   def asAnnotation: Annotation = Annotation(divOption(lh.getNumericalMean, n).orNull, lh.exactMidP(nHet))
+
+  def result(rvb: RegionValueBuilder) {
+    rvb.startStruct()
+
+    val rExpHetFreq = divNull(lh.getNumericalMean, n)
+    if (rExpHetFreq == null)
```

Comment:
Should this be null or nan? If the former, probably better to check if n is 0 explicitly which saves an allocation/boxing and is clearer. If the latter, no need to divNull anyway

Reply:
Re: NA vs NaN, I don't think we're consistent anywhere else, so this is a question to the group rather than a critique of your code!

Reply:
Let's discuss tomorrow at check-in.